### PR TITLE
Made separate functions for key width and key padding, and adjusted t…

### PR DESCRIPTION
**Pull request by henrikth93**
…he layout of the expanded keyboard.

Issue #362 (This needs to be implemented for more languages, if needed)

### Contributor checklist

- [] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Related issue

- Issue #362

### CODE REVIEW

File: Keyboards/KeyboardsBase/InterfaceVariables.swift [hunk 1]
```diff
@@ -26,7 +26,7 @@ var keyWidth = CGFloat(0)
 var letterKeyWidth = CGFloat(0)
 var numSymKeyWidth = CGFloat(0)
 var isFirstKeyboardLoad = false
-
+var disableAccentCharacters = false;
```
Comment (wkyoshida) [inline Keyboards/KeyboardsBase/InterfaceVariables.swift:29]: Oh interesting.. :thinking: just learned that semicolons are also optional in Swift :astonished: 
Not 100% on how y'all have been doing this, but we use [SwiftLint](https://github.com/realm/SwiftLint) for linting? Do we enforce linting? 

Total nit hehe - the above trailing semicolon simply caught my eye, and led me to searching a bit about SwiftLint :laughing: Appears like the community seems to prefer removing trailing semicolons? Not sure, but made me think about linting for us.

NOTE: no need to set any of this up in this PR :laughing: just commenting as something for us to think through :+1: 

CC: @andrewtavis @SaurabhJamadagni

Comment (andrewtavis) [reply Keyboards/KeyboardsBase/InterfaceVariables.swift:29]: Ya I'd say we don't need the semicolons as well, thanks @wkyoshida! Maybe we can get ourselves one of them fancy PR linting checks 🤠 Should we make an issue for it?

File: Keyboards/KeyboardsBase/KeyboardKeys.swift [hunk 2]
```diff
@@ -253,28 +253,69 @@ class KeyboardKey: UIButton {
...
+    if (usingExpandedKeyboard)
+    {
+      scalarSpecialKeysWidth = (disableAccentCharacters && keyboardState != .symbols) ? 2.2 : 1.0
+      scalarReturnKeyWidth = (disableAccentCharacters && keyboardState != .symbols) ? 2.2 : 1.0
+      if key == "ABC" || key == "АБВ" {
+        layer.setValue(true, forKey: "isSpecial")
+        widthAnchor.constraint(equalToConstant: numSymKeyWidth * 1).isActive = true
+      } else if ["delete", "#+=", "selectKeyboard", SpecialKeys.capsLock].contains(key) {
+        layer.setValue(true, forKey: "isSpecial")
+        widthAnchor.constraint(equalToConstant: numSymKeyWidth * scalarSpecialKeysWidth).isActive = true //*2 scalarSpecialKeysWidth = 1
```
Comment (andrewtavis) [inline Keyboards/KeyboardsBase/KeyboardKeys.swift:265]: We need a space between the slashes and the comment

**Review by andrewtavis — APPROVED** [review]
I'll go through and add in the minor nitpick edit to the comment, @henrikth93 :) Thanks so much for the work here! 🥳 Let's discuss what else is needed for the main issue in the sync, but I'll close #383 once I'm done with the minor edits for this 😊

**Pull request merged**